src/OpenTelemetry.Exporter.Prometheus.HttpListener/PrometheusHttpListener.cs

```diff
@@ -187,4 +192,16 @@ private async Task ProcessRequestAsync(HttpListenerContext context)
         {
         }
     }
+
+    private bool AcceptsOpenMetrics(HttpListenerRequest request)
```

Comment:
You should be able to mark this method in both this class and the middleware class static once you have marked `PometheusHeadersParser` class and its method `AcceptOpenMetrics` static.

Reply:
This is done